src/components/customHelpText/CustomHelpText.tsx

```diff
@@ -17,7 +17,7 @@ export const CustomHelpText = ({
   const inputRef = useRef(null);
   const [openState, setOpenState] = useState<boolean>(false);
   return (
-    <span className={classNames("customHelpText", className)}>
+    <div className={classNames("customHelpText", className)}>
```

Comment:
Jeg gjorde denne om til span for å få hjelpeteksten inline (se bilde), men det finnes kanskje en bedre måte å gjøre dette på. 

![image](https://user-images.githubusercontent.com/10834186/221127749-226dcfd8-ef7d-4ffe-88c0-2e9e5f80f175.png)

Ser at hjelpeteksten legger seg under på arbeidsforhold også:

![image](https://user-images.githubusercontent.com/10834186/221127858-249498bb-f8df-4fbd-aa22-066b714343a7.png)

Kanskje det er ok?

Reply:
Fikset